docker-compose.yml

```diff
@@ -2,10 +2,18 @@ version: '3'
 
 services:
 
+  database:
+    image: mysql:5.6
+    environment:
+      MYSQL_DATABASE: mysql_db
```

Comment:
@yt Let's use a proper name for database.

Reply:
@ugursogukpinar changed it to `news_feed_db`